src/emulator/kernel/src/load_self.cpp

```diff
@@ -458,15 +458,19 @@ SceUID load_self(Ptr<const void> &entry_point, KernelState &kernel, MemState &me
     strncpy(sceKernelModuleInfo->path, self_path.c_str(), 255);
 
     for (Elf_Half segment_index = 0; segment_index < elf.e_phnum; ++segment_index) {
-        emu::SceKernelSegmentInfo &segment = sceKernelModuleInfo->segments[segment_index];
-        segment.size = sizeof(segment);
-
+        // Skip non-loadable segments
         auto it = segment_reloc_info.find(segment_index);
-        if (it != segment_reloc_info.end())
-            segment.vaddr = it->second.addr;
-        else
-            segment.vaddr = 0;
+        if (it == segment_reloc_info.end())
+            continue;
 
+        if (segment_index >= 4) {
```

Comment:
Is there a named constant we can use? If not, can we add one?

Reply:
Sure